### ISSUE TITLE
app process 

### ISSUE THREAD

**Issue by dilrajkehra** [rovo89/Xposed#204]
Xposed framework is not being able to install on my phone
Samsung galaxy core 2 4.4.2 kitkat rooted
The error is
Xposed is not (yet) compatible with Android SDK version 19 or your processor architecture
(armeabi-v7a).

Warning:linker: /data/data/de.robv.android.installer/cache/app _ process has text relocations. This is wasting memory and a security risk.please fix
CANNOT LINK EXECUTABLE: cannot locate symbol
"zn7android14AndroidRuntime5startEPKcS2"
referenced by "/data/data/de.robv.android.installer/cache/app _ process"...

**Comment by rovo89** [comment]
Yeah, that's a known issue, but don't expect a fix anytime soon. The focus isn't on KitKat and older releases anymore.

**Comment by ateh2001** [comment]
Sorry guys..just want to ask..I'm  using 4.4.2 cactus...when finish installing the app(xposed)..i get this kind of error..please will somebody help me..

**Comment by Bianfuchen** [comment]
yes,solarwarez had fixed it .
you can download xposed 2.61 by solarwarez here.
https://forum.xda-developers.com/xposed/xposed-android-4-4-4-t3249895/post64063168%23post64063168
